src/providers/info.rs

```diff
@@ -208,7 +212,10 @@ impl InfoProvider for DefaultInfoProvider<'_> {
     }
 
     fn cache(&self) -> Arc<dyn Cache> {
```

Comment:
Why dies the info provider expose this?

Reply:
Some routes use it. Eg.: https://github.com/safe-global/safe-client-gateway/blob/67afbd9a4aff5c78fe0d5045d667221e4c1a4dd8/src/routes/safes/handlers/safes.rs#L60